rllib/connectors/util.py

```diff
@@ -83,14 +85,31 @@ def get_action_connectors_from_config(
 
 
 @PublicAPI(stability="alpha")
-def create_connectors_for_policy(policy: "Policy", config: TrainerConfigDict):
+def create_connectors_for_policy(
+    policy: "Policy",
+    config: TrainerConfigDict,
+    *,
```

Comment:
Is there a particular reason for having this arg-star here?

Reply:
force users to have names when calling this. otherwise you see ``create_connectors_for_policy(..., True, False)``, and it's error prone.

Reply:
Cool, thanks!